app/src/pages/organization/organizationProjectsPage/organizationProjectsPage.jsx

```diff
@@ -49,6 +52,7 @@ export const OrganizationProjectsPage = () => {
 
   const projects = useSelector(projectsSelector);
   const isProjectsEmpty = !projectsLoading && projects.length === 0;
+  const [searchValue, setSearchValue] = useState(null);
```

Comment:
Can we avoid duplicating the filter value in state?
I suppose we can use it directly from `filter` props provided via `withFilter` HOC.

Reply:
Used searchValueState from the parent component to specify what kind of empty state there is for content: 'No results found' or 'No project'

Reply:
Got it.
My point I that you can use the HOC on the page level component (parent) to rely on the `filter` prop and propagate it to the children where needed.

Reply:
Decided to leave it as is to avoid text typing debounce.